node.js

```diff
@@ -417,10 +417,10 @@ module.exports = {
 
     var latest = latestReleaseTime(agentsObj)
     var halfYearAgo = Date.now() - TIME_TO_UPDATE_CANIUSE
-
+    var elapsedMonths = Math.round((Date.now() - latest) / 30 / 24 / 60 / 60 / 1000)
```

Comment:
`30 / 24 / 60 / 60 / 1000` dublicated

Also, why you need `halfYearAgo` if you already calculated month?